jsx/StaticDataTable.js

```diff
@@ -152,25 +152,41 @@ var StaticDataTable = React.createClass({
     });
   },
   countFilteredRows: function() {
+    var useKeyword = false;
     var filterMatchCount = 0;
     var filterValuesCount = (this.props.Filter ?
-        Object.keys(this.props.Filter).length :
-        0
+            Object.keys(this.props.Filter).length :
+            0
     );
     var tableData = this.props.Data;
     var headersData = this.props.Headers;
 
+    if (this.props.Filter.keyword) {
```

Comment:
i still think there's got to be a better way of doing this rather than explicitly depending on the field explicitly being named `keyword`

Reply:
Could use `this.props.Keyword`.. but what's wrong with the filter named keyword convention (assuming it's documented)? I can't think of any situations where it would conflict with a filter explicitly named "keyword"

Reply:
generally there should be a more generic or structured way to map filters to multiple columns, but that is probably outside the scope of this PR. this is ok as an interim solution imo